frontends/web/src/style/variables.css

```diff
@@ -170,7 +170,7 @@
         --background-secondary: var(--color-softblack);
         --background-dark: var(--color-softblack);
         --background-focus: var(--color-softblack);
-        --background-tertiary: var(--color-dark);
+        --background-tertiary: var(--color-gray);
```

Comment:
`--background-tertiary` is only used for toggles and secondary buttons. This changes darkmode color of unchecked toggle background and button secondary border. I'll paste screenshots in the comments.

https://github.com/search?q=repo%3Adigitalbitbox%2Fbitbox-wallet-app%20--background-tertiary&type=code